src/chunks.rs

```diff
@@ -0,0 +1,191 @@
+use anyhow::Context;
+use std::fs::File;
+use std::io::{BufRead, BufReader, Cursor, Seek, SeekFrom};
+use std::path::Path;
+
+fn split(chunk: u64, size: u64) -> Vec<u64> {
```

Comment:
Please order the code (as in the order modules) in top-down order: top-level APIs come first, followed by things it depends on. `impl`s come after `struct` definitions.

Reply:
Done.